app-angular2/src/main/webapp/package.json

```diff
@@ -20,20 +20,21 @@
     "@angular/router": "~4.0.0",
     "angular-in-memory-web-api": "~0.3.0",
     "core-js": "^2.4.1",
-    "rxjs": "5.0.1",
+    "rxjs": "5.4.2",
     "systemjs": "0.19.40",
-    "zone.js": "^0.8.4"
+    "zone.js": "^0.8.4",
+    "keycloak-js": "~12.0.2"
```

Comment:
I wonder, couldn't we use `keycloak.js` from the server instead of hardcoding a preset version here?

Reply:
I somehow missed your comment, sorry. Well I am not sure it is a good idea. I am not really an expert in angular, but I believe this is the way how the community will use it. Maybe we should change it to `latest` so that we always test with latest realease of keycloak-js. WDYT?

Reply:
Ok, I see your point. Even though it might not be ideal (let's say we're testing a PR to the KC repo which makes a breaking change to the adapter, this solution wouldn't catch it), let's use `latest`.